src/custom/components/AccountDetails/index.tsx

```diff
@@ -118,7 +129,7 @@ export default function AccountDetails({
   const explorerLabel = chainId && account ? getExplorerLabel(chainId, account, 'address') : undefined
 
   return (
-    <>
+    <Wrapper>
```

Comment:
Adding a parent wrapper to create a parent Styled component, to then target specific child component's CSS.